app/src/main/java/com/globant/onboardingcalculatorwithkotlin/mvp/presenter/CalculatorPresenter.kt

```diff
@@ -1,16 +1,25 @@
 package com.globant.onboardingcalculatorwithkotlin.mvp.presenter
 
+import com.globant.onboardingcalculatorwithkotlin.R
 import com.globant.onboardingcalculatorwithkotlin.mvp.contracts.CalculatorContracts
+import com.globant.onboardingcalculatorwithkotlin.utils.Constants.DECIMAL_FORMAT
 import com.globant.onboardingcalculatorwithkotlin.utils.Constants.DECIMAL_POINT
 import com.globant.onboardingcalculatorwithkotlin.utils.Constants.EMPTY_CHAR
 import com.globant.onboardingcalculatorwithkotlin.utils.Constants.EMPTY_STRING
 import com.globant.onboardingcalculatorwithkotlin.utils.Constants.NUMBER_ZERO
+import com.globant.onboardingcalculatorwithkotlin.utils.Constants.OPERATOR_DIVIDE
+import com.globant.onboardingcalculatorwithkotlin.utils.Constants.OPERATOR_MULTIPLY
+import com.globant.onboardingcalculatorwithkotlin.utils.Constants.OPERATOR_PLUS
+import com.globant.onboardingcalculatorwithkotlin.utils.Constants.OPERATOR_SUBSTRACTION
+import java.text.DecimalFormat
 
 class CalculatorPresenter(
     private val model: CalculatorContracts.Model,
     private val view: CalculatorContracts.View
 ) : CalculatorContracts.Presenter {
 
+    //private val decimalFormat: DecimalFormat = DecimalFormat(DECIMAL_FORMAT)
```

Comment:
if this is not necessary, please remove it

Reply:
Done!!